packages/core/src/components/Autocomplete/Autocomplete.jsx

```diff
@@ -90,13 +92,15 @@ export class Autocomplete extends React.PureComponent {
       itemToString,
       label,
       loading,
-      onChange
+      onChange,
+      onInputValueChange
```

Comment:
Are we going to do the thing where we pass any remaining, undocumented props to Downshift?

Reply:
Added `child, {...autocompleteProps}` to our Downshift prop declaration block.